src/test/skript/tests/syntaxes/expressions/ExprBlockDataState.sk

```diff
@@ -0,0 +1,22 @@
+test "block data state" when minecraft version is "1.16":
```

Comment:
Maybe we could split these tests up? Like waterlogged for >= MC 1.13, campfire for >= 1.14, etc. to ensure that we don't miss out on covering older MC versions?

Reply:
1) Not sure why the heck I left this as 1.16 and not 1.16.1, that was a mistaken
2) For some reason tests are only running on the version that is put there, Im not sure why because the condition should be version and above, but for some odd reason the test isn't running.